test/Microsoft.ML.Core.Tests/UnitTests/TestCSharpApi.cs

```diff
@@ -254,7 +254,7 @@ public void TestCrossValidationBinaryMacro()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Differnce in Baseline files for netcoreapp 3.0")]
```

Comment:
Shouldn't we only disable this for netcoreapp3.0?

Reply:
is there any way to check that we are running things on .netcore3.0 ?